http-gateway/service/getClientConfiguration_test.go

```diff
@@ -2,44 +2,69 @@ package service_test
 
 import (
 	"context"
+	"crypto/tls"
+	"fmt"
 	"net/http"
 	"testing"
 
-	"github.com/plgd-dev/cloud/http-gateway/test"
+	"github.com/stretchr/testify/require"
+
+	"github.com/plgd-dev/cloud/grpc-gateway/pb"
+	httpgwTest "github.com/plgd-dev/cloud/http-gateway/test"
 	"github.com/plgd-dev/cloud/http-gateway/uri"
-	kitNetGrpc "github.com/plgd-dev/cloud/pkg/net/grpc"
 	rdTest "github.com/plgd-dev/cloud/resource-directory/test"
-	cloudTest "github.com/plgd-dev/cloud/test"
-	oauthTest "github.com/plgd-dev/cloud/test/oauth-server/test"
-	"github.com/plgd-dev/kit/codec/json"
-	"github.com/stretchr/testify/require"
+	"github.com/plgd-dev/cloud/test"
+	"github.com/plgd-dev/cloud/test/config"
+	testCfg "github.com/plgd-dev/cloud/test/config"
 )
 
-func TestGetClientConfiguration(t *testing.T) {
-	ctx, cancel := context.WithTimeout(context.Background(), test.TestTimeout)
+func TestRequestHandler_GetClientConfiguration(t *testing.T) {
```

Comment:
You've removed the getClientConfiguration.go, so why do we have test? Is it for well-known/ocfcloud configuration? If yes, then rename it.

Reply:
renamed test to `GetCloudConfiguration`